tests/NRedisStack.Tests/RedisFixture.cs

```diff
@@ -1,13 +1,54 @@
 using StackExchange.Redis;
+using System.Text.Json;
 
 namespace NRedisStack.Tests;
 
+public class EndpointConfig
+{
+    public List<string>? endpoints { get; set; }
+
+    public bool tls { get; set; }
+
+    public string? password { get; set; }
+
+    public int? bdb_id { get; set; }
+
+    public object? raw_endpoints { get; set; }
+
+    public ConnectionMultiplexer CreateConnection(ConfigurationOptions configurationOptions)
+    {
+        configurationOptions.EndPoints.Clear();
+
+        foreach (var endpoint in endpoints!)
+        {
+            configurationOptions.EndPoints.Add(endpoint);
+        }
+
+        if (password != null)
+        {
+            configurationOptions.Password = password;
+        }
+
+        // TODO(imalinovskiy): Add support for TLS
```

Comment:
Can we perhaps list somewhere the environments we would like to have?

I think the existing tests (which make sense to us) are a good reference as to what we need.

Everybody can suggest additions / deletions to this list

Reply:
The list of required environments depends on the current test coverage. For example, in Jedis, we need a huge list of Redis endpoints https://github.com/redis/jedis/pull/3836/files#diff-c731b96d111ab92ab2f88dcbe2e36abef53c269a71f96598463e6cc3b70b7acd and it doesn't yet cover the Sentinel and Cluster endpoints. As a first stage, I suggest identifying those endpoints and collecting them in endpoint.json file in each repo.

Reply:
Yes, this is what I meant, gather a list so we know what we need. Thanks!